client/src/components/Ticketing/event/TicketPicker.tsx

```diff
@@ -289,7 +289,7 @@ const TicketPicker = (props: TicketPickerProps): ReactElement => {
         htmlFor='time-select'
       >
         {selectedDate ? format(selectedDate, 'eee, MMM dd') : ''}
-        <span className='text-white font-bold'> - Choose time:</span>
+        <span className='text-white font-bold text-2xl'> - Choose time:</span>
       </label>
```

Comment:
For this section, it'd probably make more sense to apply the text sizing the the label, rather than just the ` - Choose time:` portion.

Reply:
I added a commit where all the UI text is now consistently enlarged, including the option boxes and option labels. Let me know what you think. Thanks!

Reply:
Also thanks for the info about mobile sizing @henrykaus